gc/base/SweepPoolManagerAddressOrderedListBase.cpp

```diff
@@ -412,6 +448,11 @@ MM_SweepPoolManagerAddressOrderedListBase::connectFinalChunk(
 				sweepState->_connectPreviousFreeEntry,
 				sweepState->_connectPreviousFreeEntrySize);
 
+		if (((MM_MemoryPoolAddressOrderedListBase *)memoryPool)->doesNeedClearCardForConnectFreeEntry()) {
```

Comment:
similar to before, for this block introduce something like connectFinalMemoryPostProcess